test/integration/targets/ec2_instance/tasks/main.yml

```diff
@@ -0,0 +1,27 @@
+---
+# A Note about ec2 environment variable name preference:
+#  - EC2_URL -> AWS_URL
+#  - EC2_ACCESS_KEY -> AWS_ACCESS_KEY_ID -> AWS_ACCESS_KEY
+#  - EC2_SECRET_KEY -> AWS_SECRET_ACCESS_KEY -> AWX_SECRET_KEY
+#  - EC2_REGION -> AWS_REGION
+#
+
+# - include: ../../../../../setup_ec2/tasks/common.yml module_name: ec2_instance
```

Comment:
It doesn't appear that the common tests are functional for this particular module - see https://github.com/ansible/ansible/blob/devel/test/integration/targets/setup_ec2/tasks/common.yml#L13 for example, which is not an error the ec2_instance module will give since a name isn't required.